LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [2021] [Derek Pisner]
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
We should remove `END OF TERMS AND CONDITIONS` and below. The rest are instructions to us.

Reply:
Thought about that Chris, but saw that the ITK folks have the exact same paragraph:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/LICENSE#L179

And decided to keep it, since I thought that the file must have been obtained as is somewhere from Apache. It also makes it clear the paragraph to include on each file (except for the year issue that we discussed). Let me know should I be wrong: happy to remove it.

Reply:
Removed in 3f66927. Force pushed.